### ISSUE TITLE
No eSim support

### ISSUE THREAD

**Issue by nestire** [srlabs/blue-merle#47]
Firmware 4.3.19 comes with eSim Support.
If I understand this correct, esim profiles come with it's own IMSI so this should work not so different to a normal Sim. But the Work flow would need to be adjusted since in the case of installing a new eSim profile you would need to use the webinterface. So for know blue merle does not support esim but the implementation should be farley straight forward 

- [ ] check how eSim support is implemented 
- [ ] integrate this in blue merle sim swap flow (webinterface + cli) Button will probably not work

**Comment by FloatingLizard** [comment]
Hi, were you able to install blu-merle with 4.3.19?  I got an error!

**Comment by nestire** [comment]
> Hi, were you able to install blu-merle with 4.3.19? I got an error!

yes using our own blue-merle build see here: https://docs.nitrokey.com/nitrowall/nw750/firmware-update 
the error that most of the times happens is that the blue-merle checks for the firmware version and if it don't know it since the build is older it needs additional confirmation see also this issue #40

**Comment by nestire** [comment]
just to give a update here: this probably is not solvable. 
My assumption in the top was wrong, there is the EID/eUICCID which if I understood it correctly, is a IMEI like Identifier for the eSIM chip which is also transferred over the network to your provider. 
So as long as there is no way to also change this there is no point in supporting eSim

**Comment by nestire** [comment]
Based on this post i would say this is a no fix 
https://discourse.osmocom.org/t/is-euiccid-changing-posible/2138/2

**Comment by nifri** [comment]
@nestire you say the EID/eUICCID is tied to the eSIM chip. For the Mudi V2 I guess this is tied to the pysical eSIM card that one has to use with the device in order to use eSIM profiles. 
If my assumption is correct the IMEI change would still work with eSIM if one also changes the pysical eSIM card at the same time and then sets up a new profile with the new card.
What do you think?

**Comment by nestire** [comment]
yes if you change the physical eSim card + ne profile that "solves" the problem but kind of also removes most of the benefits of having eSim support in the firstplace.
